odpf/common/v1/service.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package odpf.common.v1;
+
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/odpf/proton/common/v1";
+option java_multiple_files = true;
+option java_package = "io.odpf.proton.common";
+option java_outer_classname = "CommonServiceProto";
+
+
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+    info: {
+        version: "0.1";
+    };
+    external_docs: {
+        description: "Common endpoints for all services";
+    }
+    schemes: HTTP;
+};
+
+service CommonService {
+    rpc GetVersion(GetVersionRequest) returns (GetVersionResponse) {
+        option (google.api.http) = {
+            get: "/v1/version"
+        };
+    }
+}
+
+message GetVersionRequest {}
```

Comment:
should we also think of passing the client version here? So that the server can log what all client versions are interacting with the service? I do this in optimus.

Reply:
Sounds good, will make the change